drivers/SmartThings/samsung-audio/src/disco.lua

```diff
@@ -120,20 +126,24 @@ function Disco.find(deviceid, callback)
       local headers = process_response(val)
       local ip, port = headers["location"]:match(
                              "http://([^,/]+):([^/]+)") -- TODO : We need to check the xml filename for samsung audio device for ex: http://192.168.0.1:59666/rootDesc.xml
-      local id = headers["usn"]
-
+      -- local id = headers["usn"]
+ 
       -- TODO how do I know the device that responded is actually a samsung-audio device
       -- potentially will need to make a request to the endpoint
       local meta = fetch_device_metadata(headers["location"])
       local speaker_name = "samsung-audio speaker"
       local speaker_model = "unknown samsung-audio"
+      local speaker_mac = "000000000000"
```

Comment:
I am making some assumptions when I add this review comment, so if my assumptions are wrong then please correct me.

Using a MAC of all 0's as a dummy value makes it easier when you get down to the check on [Line 156](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/cdbcb7ac3e99ecff2a97482fea7222ad10e20118/drivers/SmartThings/samsung-audio/src/disco.lua#L156). But a MAC address of all 0's has semantic meaning in the ARP protocol, and so I don't think it's a good idea to use it as a placeholder value. My preference would be for this variable to be `nil` and then do an explicit `nil` check on deviceid on 156:

https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/467/files#r1073687437

Reply:
Done